operating/manage-user-access/configure-ocp-oauth.adoc

```diff
@@ -0,0 +1,27 @@
+[id="configure-ocp-oauth"]
+= Configuring {ocp} OAuth server as an identity provider
+include::modules/common-attributes.adoc[]
+:context: configure-ocp-oauth
+
+toc::[]
+
+[role="_abstract"]
+{ocp} includes a built-in OAuth server and you can use it as an authentication provider for {product-title} (RHACS).
+
+[IMPORTANT]
+====
+* Integration with {ocp} OAuth server is only available if you install the {product-title} on {ocp}. If you have installed {product-title} on Kubernetes, the user interface (UI) shows the option to configure {ocp} OAuth server, but the integration will not work.
```

Comment:
This is not applicable to Kubernetes in any way.  Theres no need to reference Kubernetes at all.